app/game/Main.hs

```diff
@@ -130,8 +133,30 @@ cliParser =
       , "t/s."
       , "(Negative values are allowed, e.g. -3 means 1 tick per 8 sec.)"
       ]
+
+  debug :: Parser (Set.Set DebugOption)
+  debug = Set.unions <$> (([Set.singleton ToggleCreative] <$ cheat) <|> many debugOption)
```

Comment:
@byorgey, this does not allow combinations of cheat and debug, like `swarm --cheat --debug=robot_id `.

I am not sure if there is a way to do it with Applicative and Alternative. If not, and we wanted to support it, I would have to put `cheatMode` back to app options and resolve it in state initialisation.

Reply:
OK, I found a way but I am not happy about it. 😅 Nicer ideas welcome.